types/test.ts

```diff
@@ -77,12 +77,8 @@ export function testOptions() {
     },
     global: {
       config: {isCustomElement: _ => true},
+      plugins: [],
     },
-    store: {
-      state: {count: 3},
-      strict: true,
-    },
-    routes: [{path: '/', component: () => SomeComponent, name: 'route name'}],
```

Comment:
Is there a reason you removed the `store` and the `state` entirely instead of putting it in `global.plugins`?

Reply:
this file just serves as a test for types, so just making sure that `global.plugins` exists would be enough

Reply:
Ahhhhh, I see. Yeah that makes sense.